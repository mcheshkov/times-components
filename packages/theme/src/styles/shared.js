const sharedStyle = {
  ThemeBody: {
    padding: 20
  }
};

export default sharedStyle;
