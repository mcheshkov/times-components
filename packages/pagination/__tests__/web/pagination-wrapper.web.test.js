import shared from "../shared.web";
import withPageState from "../../src/pagination-wrapper";

describe("web", () => {
  shared(withPageState);
});
