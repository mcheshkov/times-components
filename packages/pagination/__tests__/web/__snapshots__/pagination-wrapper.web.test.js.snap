// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1. renders inner component with page 1 1`] = `
<div>
  {
  "page": 1
}
</div>
`;
